src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+
+    public String getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```
This method should return an object of the Ball class